### PR TITLE
Improve fulltext tests

### PR DESCRIPTION
**Pull request by angrykoala**
# Description
After #3110 being reverted, this PR contains all useful changes from that PR without making any changes to the API or features:
* New fulltext tests
* `testIf` utility

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Made some changes to the GraphQL Toolbox? Or made some changes to the core library perhaps?

Check out your changes at https://neo4j-graphql-toolbox-stage-3113.surge.sh

Use for instance a local dbms hosted on [Neo4j Desktop](https://neo4j.com/download/) or a cloud-hosted [Aura Free](https://neo4j.com/cloud/aura-free/) dbms.

**Review by darrellwarde — APPROVED** [review]
Very surprised these tests didn't already exist!

File: packages/graphql/tests/integration/directives/fulltext/fulltext-index.int.test.ts [hunk 1]
```diff
@@ -0,0 +1,542 @@
...
+        const neoSchema = new Neo4jGraphQL({ typeDefs });
+        await neoSchema.getSchema();
+
+        const session = driver.session({ database: databaseName });
+
+        try {
+            await session.run(
+                [`CREATE FULLTEXT INDEX ${indexName}`, `IF NOT EXISTS FOR (n:${type.name})`, `ON EACH [n.title]`].join(
+                    " "
+                )
```
Comment (mjfwebb) [inline packages/graphql/tests/integration/directives/fulltext/fulltext-index.int.test.ts:312]:
```suggestion
                `CREATE FULLTEXT INDEX ${indexName} IF NOT EXISTS FOR (n:${type.name}) ON EACH [n.title]`
```

File: packages/graphql/tests/integration/directives/fulltext/fulltext-index.int.test.ts [hunk 2]
```diff
@@ -0,0 +1,542 @@
...
+
+            await session.run(`
+                CREATE (:${type.name} { title: "${title}" })
+            `);
+        } finally {
+            await session.close();
+        }
+    });
+
+    testIf(MULTIDB_SUPPORT)("should create index if it doesn't exist (using node label)", async () => {
```
Comment (mjfwebb) [inline packages/graphql/tests/integration/directives/fulltext/fulltext-index.int.test.ts:144]: Is there a case for multiple node labels?

**Review by mjfwebb — COMMENTED** [review]
Looks great, just a couple of comments

**Comment by angrykoala** [comment]
@darrellwarde 
Some of them already existed, but were bundled together making it hard to keep track, I split them into index and where tests

File: packages/graphql/tests/integration/directives/fulltext/fulltext-query.int.test.ts [hunk 3]
```diff
@@ -96,13 +97,7 @@ describe("@fulltext directive", () => {
...
     });
 
-    describe("Query Tests", () => {
-        // Skip if multi-db not supported
-        if (!MULTIDB_SUPPORT) {
-            console.log("MULTIDB_SUPPORT NOT AVAILABLE - SKIPPING");
-            return;
-        }
-
+    describeIf(MULTIDB_SUPPORT)("Query Tests", () => {
```
[the remark this answers is not in the document]
Comment (angrykoala) [reply packages/graphql/tests/integration/directives/fulltext/fulltext-query.int.test.ts:100]: Setting that to false skips the test. CI should compain if these tests were trying to run right?

File: packages/graphql/tests/integration/directives/fulltext/fulltext-index.int.test.ts (hunk 2 above)
Comment (angrykoala) [reply packages/graphql/tests/integration/directives/fulltext/fulltext-index.int.test.ts:144]: I... think we don't support this case, I'll add a card

**Pull request merged**